pages/docs/tutorials/message-validation.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Message validation in runtime
+description: In this tutorial, you'll learn how to validate AsyncAPI messages (events).
+
+weight: 130
+---
+
+## Introduction
+In this tutorial, you'll learn how to validate messages (events) that are sent to your AsyncAPI application.
+
+## Background context
+Message validation can be performed at both the producer and consumer levels. Message validation requires the participation of the producer, consumer, and broker. We will learn how to validate messages at the consumer level by discarding invalid messages based on the parameters provided.
+
+You will be using the [Eclipse Mosquitto](https://mosquitto.org/) broker. The MQTT protocol provides a lightweight method of messaging using a publish/subscribe model. You will also use an MQTT client that runs an MQTT library and connects to an MQTT broker over a network. Here publishers and subscribers are MQTT clients. The publisher and subscriber labels refer to whether the client is publishing or subscribing to receive messages.
+
+In the previous tutorial, you generated your application using the [AsyncAPI Generator](https://github.com/asyncapi/generator) Node.js template.
+<Remember>
+
+If you did not follow the previous tutorial and do not have an `asyncapi.yaml` file ready, then generate one using `asyncapi new --example=tutorial.yml --no-tty` command and generate server using `asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto
+cd output && npm install`

```

Comment:
```suggestion
If you did not follow the previous tutorial and do not have an `asyncapi.yaml` file ready, then generate one using `asyncapi new --example=tutorial.yml --no-tty` command and generate server using below commands:
    ```
    asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto
    cd output && npm install
    ```
```

I suggest this as the way it is done correctly it is displayed as one command, it would fail when executed in terminal